### PR TITLE
Add popover to set radius & number of POIs

### PR DESCRIPTION
**Pull request by Calli-ope**

### CODE REVIEW

File: src/lib/AdressSettings.svelte [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		poi = 1
+	}
+</script>
+
+<button
+    type="button"
+    class="trigger button-magnum"
+    use:melt={$trigger}
+>
+    <Settings2 />
```
Comment (paulpaulsen55) [inline src/lib/AdressSettings.svelte]: Maybe you should change the icon to the SlidersHorizontal icon to better match the figma

**Comment by paulpaulsen55** [comment]
Is it necessary to indicate somehow to the user that the radius is measured in km?

**Comment by OGNylux** [comment]
"POI" lieber "POIs" nennen (Plural)?

Ab einer POI-Anzahl von 9 können die letzten POIs nicht mehr angeklickt werden. Ab einer POI-Anzahl von 11 werden die letzten nicht mehr angezeigt.

**Comment by OGNylux** [comment]
Radius auf der Karte anzeigen?

File: src/lib/AdressSettings.svelte (hunk 1 above)
Comment (Calli-ope) [reply src/lib/AdressSettings.svelte]: Actually we do use the SlidersHorizontal icon atm.

Comment (paulpaulsen55) [reply src/lib/AdressSettings.svelte]: i meant this icon: https://lucide.dev/icons/sliders-horizontal

Comment (paulpaulsen55) [reply src/lib/AdressSettings.svelte]: You may also change the color of the icon-button for the map page to match the design.

image

You can do this by appendign the $$props.class prop to the class attribute on the nested Element ([REPL](https://svelte.dev/repl/6de3c939fcbf4c90a0309210c04e1c3a?version=3.17.2)). This way you can set the `button-magnum` class when needed.

Nested:
```html
<div class={"sussy " + $$props.class}></div>
```
Parent:
```html
<Nested class="bussy" />
```

**Comment by paulpaulsen55** [comment]
Wrong.

routes/+page.svelte:
```html
<AdressSettings bind:radius bind:poi class="button-magnum"/>
```
routes/map/+page.svelte:
```html
<AdressSettings bind:radius bind:poi/>
```

lib/AdressSettings.svelte:
```html
<button
    type="button"
    class={"flex items-center h-full" + $$props.class}
    use:melt={$trigger}
>
...
</button>
```

**Comment by paulpaulsen55** [comment]
> Radius auf der Karte anzeigen? https://github.com/paulpaulsen55/meedle/pull/16#issuecomment-1732355743

I think we should do a seperate branch for this, to merge the current state now for presentation.

**Review by paulpaulsen55 — APPROVED** [review]
passt

**Pull request merged**